addons/grenades/functions/fnc_throwGrenade.sqf

```diff
@@ -37,9 +37,31 @@ if (local _unit) then {
     };
 
     if (getNumber (_config >> QGVAR(flashbang)) == 1) then {
+        private _bangs = 1;
```

Comment:
`private _bangs = [_config >> QGVAR(flashbangBangs), "NUMBER", 1] call CBA_fnc_getConfigEntry;`

Reply:
No.

Reply:
cache?

Reply:
I didn't see your comments above about it. Is it because it's in `Fired` EH? Then caching at game start (like in some recent @dedmen's PRs) could be useful.

Reply:
No need to cache, unless you add `CBA_fnc_getConfigEntry` which has many if's and == checks.

Reply:
What's with the face? It's true. You don't cache single config lookups. You do it for long(ish) scripts and CBA_fnc_getConfigEntry would qualify maybe, while this currently doesn't.

Reply:
FiredEH perf isn't that important here. A millisecond or two lag on grenade throw is tolerable, you don't throw grenades that often.
Cache is beneficial for things that are looked up often. How many n-bangs do you throw in a mission? A dozen, half a dozen?
So for one cache wouldn't be that useful here. Same way the overhead doesn't matter much here.